conf/airframes/examples/bebop2_indi_gazebo.xml

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE airframe SYSTEM "../airframe.dtd">
+
+<airframe name="bebop2_indi">
+
+  <firmware name="rotorcraft">
+    <target name="ap" board="bebop2"/>
+
+    <target name="nps" board="pc">
+      <module name="fdm" type="gazebo"/>
+    </target>
+
+	  <!-- Subsystem section -->
+    <module name="telemetry" type="transparent_udp"/>
+    <module name="radio_control" type="datalink"/>
+    <module name="motor_mixing"/>
+    <module name="actuators" type="bebop"/>
+    <module name="imu" type="bebop"/>
+    <module name="gps" type="ublox"/>
+    <module name="stabilization" type="indi_simple"/>
+    <module name="ahrs" type="int_cmpl_quat">
+      <configure name="USE_MAGNETOMETER" value="TRUE"/>
+      <define name="AHRS_USE_GPS_HEADING" value="FALSE"/>
+    </module>
+    <module name="ins" type="extended"/>
+  </firmware>
+
+  <modules main_freq="512">
```

Comment:
Can you move the modules from this node to the `firmware` node ? This section is supposed to be removed in future version.

Reply:
Sure, fixed.